tests/snappi_tests/multidut/pfc/files/m2o_oversubscribe_lossless_helper.py

```diff
@@ -20,14 +19,15 @@
 
 PAUSE_FLOW_NAME = 'Pause Storm'
 TEST_FLOW_NAME = 'Test Flow'
-TEST_FLOW_AGGR_RATE_PERCENT = 30
+TEST_FLOW_AGGR_RATE_PERCENT = 35
 BG_FLOW_NAME = 'Background Flow'
-BG_FLOW_AGGR_RATE_PERCENT = 25
```

Comment:
@amitpawar12 @selldinesh @ysmanman for viz.

Reply:
In the description:
"in multi_lossless helper: We reduce the background traffic rate, and raise the rate for lossless traffic, thus keeping the overall rate still same. This allows for accounting the speed of backplane."
Earlier lossless + lossy traffic is 30% + 25% = 55% from each ingress.
Now lossless + lossy traffic is 35% + 22.5% = 57.5% from each ingress.

The traffic rate is not same. 

@sdszhang , @vmittal-msft

Reply:
Yes, @amitpawar12 . That was intentional. We had discussions, and since lossless will get paused anyways, it can be higher, if we want.